src/app/app.module.ts

```diff
@@ -19,7 +26,15 @@ import { FooterComponent } from './footer/footer.component';
     RegformModule,
     TrainingdescriptionModule,
     HomepageModule,
-    BrowserAnimationsModule
+    BrowserAnimationsModule,
+    HttpClientModule,
+    TranslateModule.forRoot({
+      loader: {
```

Comment:
Create file like 'app.i18n.config.ts' and move this object in there. Along with createTranslateLoader function.